src/django_subatomic/db.py

```diff
@@ -55,14 +55,27 @@ def transaction_if_not_already(*, using: str | None = None) -> Generator[None]:
     """
     Create a transaction if one isn't already open.
 
+    Can be used as a decorator or a context manager.
+
     Use of this hints at code which lacks control over the state it's called in.
 
-    Suggested alternatives:
+    Note:
+        This has a bit of a clunky name. This is a deliberate attempt to
+        discourage its use. It reflects that using this function is a
+        bit of a clunky thing to do.
```

Comment:
Note: This is lifted from other internal docs. There's probably a better way of putting this. Suggestions welcome.

Reply:
I've gone with:

```diff
     Note:
         This has a bit of a clunky name. This is a deliberate attempt to
-        discourage its use. It reflects that using this function is a
-        bit of a clunky thing to do.
+        discourage its use. It acts as a code-smell to highlight that places
+        which use it may need further work to achieve full control over how
+        transactions are managed.
```